domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Child.java

```diff
@@ -48,6 +60,10 @@ public Integer getNumberOfChildren() {
         return Optional.ofNullable(numberOfChildrenLivingWithYou);
     }
 
+    public String getAgeGroupTypeDescription() {
```

Comment:
Please remove. Content provided should take care of it.

Reply:
same as above.